units/systemd-udevd.service.in

```diff
@@ -21,7 +21,10 @@ Sockets=systemd-udevd-control.socket systemd-udevd-kernel.socket
 Restart=always
 RestartSec=0
 ExecStart=@rootlibexecdir@/systemd-udevd
-MountFlags=slave
 KillMode=mixed
 WatchdogSec=3min
 TasksMax=infinity
+MountFlags=slave
+MemoryDenyWriteExecute=yes
+RestrictRealtime=yes
+RestrictAddressFamilies=AF_UNIX AF_NETLINK
```

Comment:
The wget thing can't really work, as hw plugs would race against network configuration...

I am fully aware though that this might break stuff, but it's stuff I'd claim was actually broken. I think a device manager should not be able to talk to the Internets, hence I changed the default. But we should of course document this in NEWS, so that people are aware, and distros being too afraid of this change can revert it locally.

I'll add a blurb about this to the NEWS file as part of this patch